Javascript/UIGen.js

```diff
@@ -100,10 +125,26 @@ function generateSidebar(guideInfo)
 
     //Body
     var bodyDiv = document.createElement("div");
-    bodyDiv.innerHTML = "<br>" + guideInfo["Guide"] + "<br><br><br>";
+    bodyDiv.innerHTML = "<br>" + guideInfo["Guide"] + "<br><br>";
+    bodyDiv.innerHTML += "-------------------------------------------<br>";
+    bodyDiv.innerHTML += "&nbsp;&nbsp;&nbsp;Did we get this type wrong?&nbsp;&nbsp;<br><br>";
     bodyDiv.classList.add("bodyDiv");
     sb.appendChild(bodyDiv);
 
+    //Error Button
```

Comment:
I really like this feature, we needed more exception handling according to the professor.